Shifty.App/Components/UnusedTickets.razor

```diff
@@ -0,0 +1,95 @@
+@namespace Components
+@using System.ComponentModel.DataAnnotations
+@using Shifty.App.Services
+@using Shifty.Api.Generated.AnalogCoreV1
+@using Shifty.Api.Generated.AnalogCoreV2
+@using Shifty.App.DomainModels
+@using Shifty.App.Shared
+@using Shared
+@using LanguageExt.UnsafeValueAccess
+@inject IUnusedTicketsService _unusedTicketsService
+@inject ISnackbar Snackbar
+
+<MudContainer MaxWidth="MaxWidth.Medium" Style="margin-top: 20px;">
+    <MudDataGrid
+        T="UnusedTicket"
+        Items="@Items"
+        Height="500px"
+        FixedFooter>
+        <ToolBarContent>
+            <MudText Typo="Typo.h6">Unused Tickets</MudText>
+            <MudSpacer />
+            @if (_loading)
+            {
+                <LoadingIndicator Height="64px" />
+            }
+            <MudDateRangePicker DateRange="@_queryDateRange" Editable=true DateRangeChanged="LoadUnusedTickets"/>
+        </ToolBarContent>
+        <Columns>
+            <PropertyColumn Property="x => x.ProductId" Title="Product Id" />
+            <PropertyColumn Property="x => x.ProductName" Title="Product Name" AggregateDefinition="_footerLabel"/>
+            <PropertyColumn Property="x => x.TicketsLeft" Title="Unused Tickets" AggregateDefinition="_ticketsLeftSum" />
+            <PropertyColumn Property="x => x.UnusedPurchasesValue" Title="Unused value (DKK)" AggregateDefinition="_valueLeftSum">
+                <CellTemplate>
+                    @context.Item.UnusedPurchasesValue.ToString("0.00")
+                </CellTemplate>
+            </PropertyColumn>
+        </Columns>
+        <NoRecordsContent>No matching records found</NoRecordsContent>
```

Comment:
When would this ever happen?

Reply:
Default query values are set to January 1st to December 31st in the current year. If the user opens this in the beginning of the year, i.e. no tickets have been purchased but unused for this given period exists, then this will show.

Likewise, if the user is putting in "invalid" query values such as querying unused tickets for year 2025, a from date before the to date etc. So even in production, this can happen in multiple circumstances.

Reply:
Done